electricpy/__init__.py

```diff
@@ -59,17 +59,22 @@ def tcycle(ncycles=1, freq=60):
     -------
     t:          float
                 Total time for *ncycles*
+
+    Examples
+    --------
+    >>> import electricpy as ep
+    >>> ep.tcycle(1, freq=60) #Value of ncycles=1 & freq=60
+    0.01666667
+    >>> ep.tcycle(1, freq=50) #Value of ncycles=1 & freq=50
+    0.02
     """
     # Condition Inputs
     ncycles = _np.asarray(ncycles)
     freq = _np.asarray(freq)
     # Evaluate the time for ncycles
     time = ncycles / freq
     # Return
-    if len(time) == 1:
```

Comment:
Hmm...

Instead of just removing this logic, how about we wrap it in a Python `try`/`except` block that will let us catch that `TypeError` exception and handle it appropriately. Take a look at my example below, what do you think?

```python
try:
    if len(time) == 1:
        return time[0]
    else:
        return time
except TypeError:
    return time
```

Reply:
Thank you for the suggestion, I think it's reasonable to add try/except,
Let me update the code